src/Components/Server/src/Circuits/CircuitHandleRegistry.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components.Lifetime;
+using Microsoft.AspNetCore.Components.Routing;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.JSInterop;
+
+namespace Microsoft.AspNetCore.Components.Server.Circuits
+{
+    internal class CircuitHandleRegistry : ICircuitHandleRegistry
```

Comment:
```suggestion
    internal sealed class CircuitHandleRegistry : ICircuitHandleRegistry
```

Reply:
this type's like a glorified dictionary accessor. Do we really need it? (I haven't seen the rest of the PR as yet.)